Create ui-c8y-1018-503-85-label-change-aim-iam-6039.md (#2014)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1018-503-85-label-change-aim-iam-6039.md

---
date: ""

> title: label change AIM -> IAM (#6039)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Update label from AIM to IAM in user interface
```

> title: Fixed typo in IAM (AIM -> IAM).

Reviewer: ```suggestion
title: Fixed typo in Single sign-on page
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix

> component:

Reviewer 2: isn't this ? 
component:
  - value: q3kclF6pO
    label: Authentication

  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-55205
version: 1018.503.85
---

> label change AIM -> IAM (#6039)

Author: The Claude AI suggested to use the following body:
```suggestion
As part of ongoing improvements to align with current terminology, the label "AIM" has been updated to "IAM" throughout the user interface. This change provides clarity and consistency by using the more widely recognized term "IAM" (Identity and Access Management). Users will now see "IAM" instead of "AIM" when interacting with identity and access management related features and settings. This label update does not impact any existing functionality or configurations.
```

> The typo has been corrected. IAM stands for Identity and Access Management.

Reviewer: ```suggestion
In the **Single sign-on** page, a typo  has been fixed (AIM -> IAM). IAM stands for Identity and Access Management.
```
